django/contrib/sites/checks.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+from django.core.checks import Error
+
+
+def check_site_id_type(**kwargs):
+    errors = []
+    if hasattr(settings, 'SITE_ID'):
+        if not isinstance(settings.SITE_ID, int):
+            errors.append(
+                Error(
+                    "``SITE_ID`` must be of type int, got %s" % type(settings.SITE_ID),
+                    obj="sites.E003"
```

Comment:
I would use `sites.E101` to separate them from checks related with `CurrentSiteManager`.